weld/conf.rs

```diff
@@ -27,3 +30,12 @@ pub fn parse_memory_limit(s: CString) -> i64 {
         Err(_) => DEFAULT_MEMORY_LIMIT,
     }
 }
+
+/// Parses the optimization passes. Returns the default if the string cannot be parsed.
+pub fn parse_optimization_passes(s: CString) -> Vec<String> {
+    let s = s.into_string().unwrap();
+    match s.as_ref() {
+        "" => DEFAULT_OPTIMIZATION_PASSES.to_vec().iter().map(|e| e.to_string()).collect(),
```

Comment:
Maybe DEFAULT_OPTIMIZATION_PASSES should be a static Vec<String> and we should just clone it. Not sure whether that's easier.